index.js

```diff
@@ -69,33 +69,4 @@ var mzKeys = [
 ]
 mzKeys.forEach(function(key){
   exports[key] = mzfs[key]
-})
-
-
-// Resolve fs-extra streams as Promise for array
-var streamKeys = [
-  'walk'
```

Comment:
What was the reasoning for removing this? (i.e. resolving walk as a Promise instead of a Stream?).   I guess that, other than this, you could use fs-promise as a drop-in for fs and still use callbacks if you wanted...